### PR TITLE
Misc/stl cameramount single

### PR DESCRIPTION
**Pull request by khrdrgb96**
Created STL model of single camera mount

### CODE REVIEW

File: hardware/CamRobot/RaspiMount.scad [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        cylinder(h=1, r=camera_hole_r);
+
+}
+
+}
+
+half();
+mirror([1, 0, 0]){
+half();
+}
```
Comment (poifull10) [inline hardware/CamRobot/RaspiMount.scad]: please add `<LF>`

**Review by poifull10 — CHANGES_REQUESTED** [review]
.DS_Store should be deleted. Please add specific files in your env into `~/.config/git/ignore` .

File: hardware/CamRobot/RaspiMount.scad (hunk 1 above)
Comment (khrdrgb96) [reply hardware/CamRobot/RaspiMount.scad]: What dose it mean?

Comment (poifull10) [reply hardware/CamRobot/RaspiMount.scad]:
```suggestion
}

```

File: hardware/CamRobot/RaspiMount.scad [hunk 2]
```diff
@@ -0,0 +1,53 @@
+$fa = 1;
+$fs = 0.01;
+
+camera_hole_r = 0.1;
+camera_hole_pitch_w=2.1;
+camera_hole_pitch_h=1.25;
+upper_hole_h = 1;
```
Comment (poifull10) [inline hardware/CamRobot/RaspiMount.scad]: How about make a module interface for reusability?
That is,  how about add `module RaspiMount(camera_hole_r, camera_hole_pitch_w, camera_hole_pitch_h, upper_hole_h, $fa, $fb) { ... ` ?
And you can use the default value for recommendation. `module RaspiMount(camera_hole_r = 0.1, ... `

**Review by poifull10 — APPROVED** [review]
LGTM, Great work!

**Pull request merged**